### DISCUSSION TITLE
does one need to open an issue to add a beta version to cask?

### DISCUSSION THREAD

**Discussion by GreenBattery** [Homebrew/discussions#3532] [Casks] [answered]
I wanted to try out amethyst (tiling window manager) beta version because it supports custom layouts but I couldn't find it in brew. 

I wonder if I need to open an issue for someone more knowledgeable to add this ?

**Comment by carlocab** [comment, marked as answer]
No, we close issues that request the addition of casks.

From the [README](https://github.com/Homebrew/homebrew-cask#requests):

> - Issues requesting new casks will be closed. If you want a cask added to the main repositories, [submit a pull request](https://github.com/Homebrew/homebrew-cask/blob/HEAD/CONTRIBUTING.md#adding-a-cask).
